### DISCUSSION TITLE
CAUB QUESTION

### DISCUSSION THREAD

**Discussion by CGRmags** [3kh0/ext-remover#936] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
If you do CAUB on SKIOVOX does it work if you sign out and come back to your school account?

**Comment by CGRmags** [comment]
@AshtonDavies Pls help me answer this

**Comment by AshtonDavies** [comment, marked as answer]
Yeah I think so. Kiosk applications are essentially small user profiles with data storing, so I believe caub should still work when you get back in.

**Reply by CGRmags** [reply]
So is the rumors about skiovox not working on 121 true? because I think I have caub on so I was wondering

**Reply by AshtonDavies** [reply]
> So is the rumors about skiovox not working on 121 true? because I think I have caub on so I was wondering
> 

Maybe it updated outside of the kiosk application.

**Comment by CGRmags** [comment]
wydm and if you dont its ok but if you here anything could you tell me

On Fri, Feb 16, 2024 at 10:25 AM Ashton Davies ***@***.***>
wrote:

> So is the rumors about skiovox not working on 121 true? because I think I
> have caub on so I was wondering
>
> Maybe it updated outside of the kiosk application.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/936#discussioncomment-8495850>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDXUJBEXMX6PBXMCXJSFGBTYT6QBBAVCNFSM6AAAAABDMGIV2OVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DIOJVHA2TA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>

-- 
NOTICE: This email message and/or its attachments may contain information 
that is confidential or restricted. It is intended only for the individuals 
named as recipients in the message. If you are NOT an authorized recipient, 
you are prohibited from using, delivering, distributing, printing, copying, 
or disclosing the message or content to others and must delete the message 
from your computer. If you have received this message in error, please 
notify the sender by return email.

**Comment by v4pl** [comment]
Skiovox is patched on 121 but I did caub 2 days before the update. (;
